### PR TITLE
[#175716281] Manage access permissions

### PR DESCRIPTION
**Pull request by miljan-kolosek**

### CODE REVIEW

File: app/controllers/stories_controller.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class StoriesController < ApplicationController
   before_action :set_story, only: [:edit, :show, :update, :destroy]
 
   def index
+    @users = User.all.decorate
```
Comment (tamara-kolosek) [inline app/controllers/stories_controller.rb:7]: maybe except admins

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -15,10 +16,24 @@ def destroy
...
+  def add_person
+    @user.role = "editor"
+    @user.save
+
+    redirect_to stories_path
+  end
+
+  def remove_person
+    @user.role = "author"
+    @user.save
```
Comment (tamara-kolosek) [inline app/controllers/users_controller.rb:28]: isn't the requirement that admin set users permission for separate story.

This way, when user gets auditor role, all the stories will be hidden from him, not just targeted one,  right? and vice versa

**Review by tamara-kolosek — CHANGES_REQUESTED** [review]

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (miljan-kolosek) [reply app/controllers/users_controller.rb:28]: The requirement is that admin set users permission to be on the list for access to stories and that is implemented.
When user gets auditor role, all stories will be hidden from him except his stories,and thats the logic from previous tickets.

File: app/controllers/stories_controller.rb (hunk 1 above)
Comment (miljan-kolosek) [reply app/controllers/stories_controller.rb]: Yes, I fix that.

**Review by tamara-kolosek — APPROVED** [review]

**Pull request merged**